### ISSUE TITLE
The parser and tagger pipes are missing after I trained my custom ner.

### ISSUE THREAD

**Issue by kasabaman** [explosion/spaCy#5192]
## How to reproduce the behaviour

import spacy
import random

TRAIN_DATA = [#I have to remove the train data]
                

def train_spacy(data,iterations):
    TRAIN_DATA = data
    nlp = spacy.blank('en')  # create blank Language class
    # create the built-in pipeline components and add them to the pipeline
    # nlp.create_pipe works for built-ins that are registered with spaCy
    if 'ner' not in nlp.pipe_names:
        ner = nlp.create_pipe('ner')
        nlp.add_pipe(ner, last=True)
       

    # add labels
    for _, annotations in TRAIN_DATA:
         for ent in annotations.get('entities'):
            ner.add_label(ent[2])

    # get names of other pipes to disable them during training
    other_pipes = [pipe for pipe in nlp.pipe_names if pipe != 'ner']
    with nlp.disable_pipes(*other_pipes):  # only train NER
        optimizer = nlp.begin_training()
        for itn in range(iterations):
            print("Statring iteration " + str(itn))
            random.shuffle(TRAIN_DATA)
            losses = {}
            for text, annotations in TRAIN_DATA:
                nlp.update(
                    [text],  # batch of texts
                    [annotations],  # batch of annotations
                    drop=0.2,  # dropout - make it harder to memorise data
                    sgd=optimizer,  # callable to update weights
                    losses=losses)
            print(losses)
    return nlp

prdnlp = train_spacy(TRAIN_DATA, 10)

# Save our trained Model
modelfile = input("Enter your Model Name: ")
prdnlp.to_disk(modelfile)

#Test your text
test_text = input("Enter your testing text: ")
doc = prdnlp(test_text)
for ent in doc.ents:
    print(ent.text, ent.start_char, ent.end_char, ent.label_)

I used above code to train my custom model.
After I created my custom model, the "parser" and "tagger" pipes were missing already. How to put it back together with my custom ner model.

**Comment by svlandeg** [comment]
It looks like you never actually had a parser or tagger as part of the pipeline? You created a blank model (which does only tokenization) and added an `ner` pipe. You can double check this by printing `nlp.pipe_names` before and after you run the IO.

If you want a pretrained tagger and/or parser, you could use a different model like `en_core_web_lg` instead of the blank one, see also https://spacy.io/models/en. Note that this pretrained model will come with a built-in `ner` component. You can then either remove that pretrained `ner` component and add a newly trained one to the pipeline like in your original code snippet, or you could consider retraining the existing component, see also here: https://spacy.io/usage/training#ner

I'll close this as there is no action for us to follow up on. For further general usage questions, please consider using StackOverflow as we try to keep this tracker focused mainly on bug reports and feature requests. Thanks !
